packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -501,8 +501,7 @@ export function TransferPanelMain({
   useEffect(() => {
     setFrom(externalFrom)
     setTo(externalTo)
-    setQueryParams({ l2ChainId: l2.network.id })
```

Comment:
Doesn't seem like we need it anymore. I'm running more tests for network switching with all possible combinations but looking good so far.

Reply:
Does it work properly if we don't make any changes? I'd rather do that, given that we'll be getting rid of this soon.